### DISCUSSION TITLE
Question: Global default config file location

### DISCUSSION THREAD

**Discussion by MadFisa** [fortran-lang/fortls#409] [Q&A]
Hi,

The Qusetion is pretty much in the title. I could find the location of project-specific config files in the documentation, but couldn't figure out where the global defaults are stored.
Where can I find the global config file?

Thanks

**Comment by gnikit** [comment]
There is no global configuration file. The configuration file is a serialised version of the CLI, so it matches 1:1 the options. There is an issue open for displaying the fortls options (cli+config) when you spin up the server https://github.com/fortran-lang/fortls/issues/348
